tests/test_circuit.py

```diff
@@ -179,55 +177,6 @@ def test_add_gate(self):
         assert qc.gates[6].targets == [5]
         assert qc.gates[5].arg_value == 1.570796
 
-        # Test Exceptions  # Global phase is not included
```

Comment:
These tests are removed because they are too detailed and only checking `ValueError` without the error message also does not correctly check if the exception raised is the correct one. In addition, in the new gate class structure, the input control is moved to each class.